ipamd/datastore/data_store.go

```diff
@@ -45,6 +46,27 @@ var ErrUnknownPod = errors.New("datastore: unknown pod")
 // ErrUnknownPodIP is an error where pod's IP address is not found in data store
 var ErrUnknownPodIP = errors.New("datastore: pod using unknown IP address")
 
+var (
+	enis = prometheus.NewGauge(
+		prometheus.GaugeOpts{
+			Name: "eni_allocated",
+			Help: "The number of eni allocated",
+		},
+	)
+	totalIPs = prometheus.NewGauge(
+		prometheus.GaugeOpts{
+			Name: "total_ip_addresses",
+			Help: "The total number of ip addresses",
+		},
+	)
+	assignedIPs = prometheus.NewGauge(
+		prometheus.GaugeOpts{
+			Name: "assigned_ip_addresses",
+			Help: "The number of ip addresses assigned",
```

Comment:
please capitalize initialisms IP and ENI in Help text.  also eni should be plural ENIs